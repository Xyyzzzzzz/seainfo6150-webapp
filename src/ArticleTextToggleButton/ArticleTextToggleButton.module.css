.toggleButton{
    background-color: rgb(200, 0, 0);
    color: white;
    padding: 10px;
    border-radius: 10px;
    margin-top: 10px;
}