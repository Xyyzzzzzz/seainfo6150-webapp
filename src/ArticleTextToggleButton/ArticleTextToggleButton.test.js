import React from "react";
import ArticleTextToggleButton from "./ArticleTextToggleButton";

describe("ArticleTextToggleButton tests", () => {
    it("renders correctly after show more button is clicked", () => {
        const { container, getByText } = render(
            <ArticleTextToggleButton buttonText={'Show more'} />
        );
        const button = getByText("Show more");
        fireEvent.click(button);
        expect(container).toMatchSnapshot();
    });
});
