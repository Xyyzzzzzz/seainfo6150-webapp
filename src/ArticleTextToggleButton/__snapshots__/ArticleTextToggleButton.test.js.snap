// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticleTextToggleButton tests renders correctly 1`] = `
<div>
  <button
    class="toggleButton"
  />
</div>
`;

exports[`ArticleTextToggleButton tests renders correctly after show more button is clicked 1`] = `
<div>
  <button
    class="toggleButton"
  >
    Show more
  </button>
</div>
`;

exports[`ArticleTextToggleButton tests renders item correctly 1`] = `undefined`;
